src/main/java/com/salesforce/selenium/support/findby/ShadowPathHelper.java

```diff
@@ -0,0 +1,30 @@
+package com.salesforce.selenium.support.findby;
+
+/***
+ * 
+ * Provide helper function to manage Shadow Path
+ * 
+ * @author ytao
+ * 
+ */
+public class ShadowPathHelper {
+	public static String getShadowQueryString(String shadowPath) {
```

Comment:
Please rename to e.g. script2ShadowPath(..) to make it more apparent what direction the conversion flows.